<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GynNav</class>
 <widget class="QWidget" name="GynNav">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>956</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="ctkCollapsibleButton" name="verifyCollapsibleButton">
     <property name="text">
      <string>Verify</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="ctkCollapsibleButton" name="segmentCollapsibleButton">
     <property name="text">
      <string>Load &amp;&amp; Segment</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="qMRMLSegmentEditorWidget" name="SegmentEditorWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QPushButton" name="stylusVisButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="referenceVisButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="referenceVisLabel">
       <property name="text">
        <string>Reference</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="stylusVisLabel">
       <property name="text">
        <string>Stylus</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="probeVisButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="probeVisLabel">
       <property name="text">
        <string>US Probe</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="ctkCollapsibleButton" name="registerCollapsibleButton">
     <property name="text">
      <string>Register</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4"/>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="ctkCollapsibleButton" name="captureCollapsibleButton">
     <property name="text">
      <string>Capture &amp;&amp; Identify Surface</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSegmentEditorWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSegmentEditorWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
